src/test_report.py

```diff
@@ -26,7 +29,37 @@ def __init__(self, configs, args):
         self._db_config = configs['db_configs'][args.db_config]
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
-        self._logger = Logger("config/logger.conf", "test_report")
```

Comment:
It looks like the rebase removed some previous changes.  Please fix this again.

Reply:
yes. I noticed this and it is already updated.